pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -439,23 +451,121 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 	}
 	//#endregion
 
-	//#region Postgres DeploymentConfig
-	postgresqlDeploymentConfig, err := newPostgresqlDeploymentConfig(instance)
+	//#region MIGRATION from old resources to new ones
```

Comment:
This is the migration code

Reply:
Hi @aliok, I also asked this question in @grdryn's pr. About the migration code. Should we add a version to the comments or something similar so we know that in a few versions we can remove it?

Reply:
Makes sense. Which PR is that?

Reply:
Done now